package.json

```diff
@@ -115,17 +115,16 @@
     "@babel/core": "^7.8.4",
     "@babel/preset-env": "^7.8.4",
     "@elastic/elasticsearch": "^7.10.0",
-    "body-parser": "^1.19.0",
-    "@hapi/hapi": "^18.4.1",
+    "@hapi/hapi": "^20.1.2",
```

Comment:
The "POST" handler test in test/instrumentation/modules/hapi.js *hangs* with `@hapi/hapi@18.x` and node 16. Let's just move to testing with the latest hapi (which *does* mean only testing with node v12 and later). The TAV tests cover the rest.